### PR TITLE
feature/achievment api

### PR DESCRIPTION
**Pull request by ArmantasPik**
Add requirement of implementing achievment system

### CODE REVIEW

File: EncountifyAPI/Startup.cs [hunk 1]
```diff
@@ -109,7 +117,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
-            app.UseMiddleware<StatisticsMiddleware>();
+            //app.UseMiddleware<StatisticsMiddleware>();
```
Comment (0rl1s) [inline EncountifyAPI/Startup.cs:120]: I think it would be better to delete all comments like this one.

**Review by AndrejJazevic — COMMENTED** [review]
Looks very good! A lot of work has been done. There is nothing more to add. Good job!

File: Encountify/Views/RegisterPage.xaml.cs [hunk 2]
```diff
@@ -54,6 +56,10 @@ private async void OnRegisterClicked(object sender, EventArgs e)
                         user.Password = Password.Text;
                         user.Email = Email.Text;
                         RegisterUser(user);
+                        await DisplayAlert("Congratulations!", "You got an achievment, Welcome to Encountify!", "OK");
```
Comment (0rl1s) [inline Encountify/Views/RegisterPage.xaml.cs:59]: This achievement message only for new users?

**Review by 0rl1s — APPROVED** [review]
Very good job. Thank you for your spend time and energy!

File: Encountify/Views/RegisterPage.xaml.cs (hunk 2 above)
Comment (ArmantasPik) [reply Encountify/Views/RegisterPage.xaml.cs:59]: Yes

File: EncountifyAPI/Startup.cs (hunk 1 above)
Comment (ArmantasPik) [reply EncountifyAPI/Startup.cs:120]: I do not want to delete it, I just commented out the middleware just for now, because it was glitching.

**Review by AndrejJazevic — APPROVED** [review]

**Pull request merged**